x-pack/legacy/plugins/reporting/config.ts

```diff
@@ -140,7 +140,9 @@ export async function config(Joi: any) {
     }).default(),
     encryptionKey: Joi.when(Joi.ref('$dist'), {
       is: true,
-      then: Joi.string(),
+      then: Joi.string()
+        .allow(null)
+        .default(null),
```

Comment:
This one gives me a bit of pause: could this break existing installs?

Reply:
I'm a little uncertain that the original code made it so the warning about encryption key would never get logged. Let me dig further and see if that's a valid standalone issue

Reply:
https://github.com/elastic/kibana/commit/31ce503c0af#diff-535b0c13b6ce00e3e031945f4a6fd9f6R131

Reply:
Sorry, it looks like I was mistaken about a change needed here. I reverted this change